### ISSUE TITLE
What to do with the BQM

### ISSUE THREAD

**Issue by gdarai** [gdarai/SpaceRace#18]
Better Questing is the Quest Book in the game...
This is the moment where I/You/We should think about what and how to incorporate in it.
Currently there are following concepts:

Overview guide chapter (Done, All Stub)
 - Guides user from planet to planet, not really quests, more like place for important messages.
Quest chapters for technology steps forward (Done, All Stub)
 - The traditional quests. Do this Craft that... guide through simple tasks
Chapter with Markets (Not Done, Only Examples)
 - How to get resources without mining or food without cooking. How to overcome the pain if we have some trouble? W can BUY what we need.
Achievements (Not Even In yet)
 - Hoarding Quests for every part of the game. Is it easy to get 1000 Gravel? Sure, but before Crusher? Player can show his spirit.

Now this is what I had/have in mind... and in the game is it in the current state so you can go through it and see it in live. The big question is: Is this what we want? How do we want this book to work? Is this concept ok? What would you improve and how?

**Comment by kotja3** [comment]
First make BQM usable at every GUI scale. Normaly I use Large and it doesn't fit on my monitor .

**Comment by gdarai** [comment]
Ok... that is a new one... do you know how to do it? Because I don't. Right now, there is hardcoded position for every quest, that is why I used the smalles scale so it would fit for every GUI scale... but I am not sure if it can be somehow set to scalable.

But first thing I can think of is to at least test it on every GUI scale so I see how it looks... if you mean this.

**Comment by kotja3** [comment]
Fruits quest doesn't recognise fruits except lemons.

**Comment by gdarai** [comment]
You are right... I set up wrongly the item stacks for the others. Thanks. It is fixed now.

**Comment by gdarai** [comment]
I was also thinking about a chapter with cheats... something like: If everything went wrong... this may save you.
